src/micro_http/src/common/headers.rs

```diff
@@ -179,6 +196,12 @@ impl Headers {
         self.expect
     }
 
+    /// Returns the `Accept` header `MediaType`.
+    #[allow(unused)]
```

Comment:
This allow should not be needed as the function is pub.

Reply:
Done.